Licensing_Program/software_dmv.py

```diff
@@ -1,292 +1,32 @@
 #! /usr/bin/env python3
 
-import argparse
-import os
-import pprint
-import re
 
 import config_handling
```

Comment:
Might as well remove this import also. utils/load_config.py might be a working replacement.